packages/react-scripts/scripts/start.js

```diff
@@ -106,11 +106,13 @@ checkBrowsers(paths.appPath)
       if (err) {
         return console.log(err);
       }
-      if (isInteractive) {
+      if (isInteractive && !process.env.NO_CLEAR_CONSOLE) {
```

Comment:
there is a way to achieve the same functionality right now, but not too pretty https://github.com/facebook/create-react-app/issues/2495#issuecomment-344537005

Reply:
thanks again! 😄

Reply:
watch https://github.com/facebook/create-react-app/pull/3288 for existing PR with the same functionality if you want it using env variable :)